test/models/patient_test.rb

```diff
@@ -152,6 +152,18 @@ class PatientTest < ActiveSupport::TestCase
       assert_equal summary[:pledged].count, 1
       assert_equal summary[:sent].count, 1
     end
+
+    it "should support a different start of week" do
+      # if the week did NOT start on thursday then patients would not be counted
+      Config.create config_key: :start_of_week,
+                    config_value: {options: ['Wednesday']}
+      @patient.update fund_pledged_at: Date.parse("January 20, 1973"), fund_pledge: 300
+        shaped_patient = patient_to_hash @patient
```

Comment:
please forgive me for this suggestion, I've been writing a lot of python lately and the extra two spaces makes me think this is a continuation of L160, haha. 

```suggestion
      shaped_patient = patient_to_hash @patient
```

Reply:
Whoops, that's a typo, good catch!

Reply:
And I didn't even use shaped_patient after all, removed it completely.